app/models/SicStore.scala

```diff
@@ -61,6 +70,7 @@ object Sector {
   implicit val format: Format[Sector] = (
     (__ \ "code").format[String] and
     (__ \ "name").format[String] and
+    (__ \ "nameCy").format[String] and
```

Comment:
Could you give this an empty string default. Also, add a subtask to remove the default later. This is to prevent breaking people with current journeys.